src/hud_elements.cpp

```diff
@@ -351,11 +351,7 @@ void HudElements::vram(){
         ImGui::TableNextColumn();
         ImGui::TextColored(HUDElements.colors.vram, "VRAM");
         ImGui::TableNextColumn();
-        // Add gtt_used to vram usage for APUs
-        if (cpuStats.cpu_type == "APU")
-            right_aligned_text(HUDElements.colors.text, HUDElements.ralign_width, "%.1f", gpu_info.memoryUsed + gpu_info.gtt_used);
-        else
-            right_aligned_text(HUDElements.colors.text, HUDElements.ralign_width, "%.1f", gpu_info.memoryUsed);
+        right_aligned_text(HUDElements.colors.text, HUDElements.ralign_width, "%.1f", gpu_info.memoryUsed + gpu_info.gtt_used);
```

Comment:
We only wants to combine gpu memory and gtt on APUs, not on dGPUs

Reply:
Oh oops! I thought ggt_used would always be 0 except when the card has an APU

Reply:
I was mainly confused why amdgpu was modifying the global `cpuStats` object, shouldn't something like this be stored in `gpu_info`?

Reply:
Yes it would be more fitting in `gpu_info`